webv2/profile/profile.go

```diff
@@ -179,7 +190,7 @@ func setConnectionProfile(isSource bool, sessionState session.SessionState, req
 	} else {
 		req.ConnectionProfile.Profile = &datastreampb.ConnectionProfile_GcsProfile{
 			GcsProfile: &datastreampb.GcsProfile{
-				Bucket:   bucket,
+				Bucket:   strings.ToLower(sessionState.Conv.Audit.MigrationRequestId),
```

Comment:
the bucket name is set as the request id ? Is this the case even if the name is passed ?